articles/azure/azs-how-create-vm-image-powershell.md

```diff
@@ -0,0 +1,171 @@
+---
+title: How to create a custom image from a VM on Azure Stack using PowerShell | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using Powershell
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM using PowerShell
```

Comment:
use "- PowerShell" here to keep the title shorter as it is displayed in an area with less space:
Create a custom image from a VM - PowerShell